Adjust sepolicy change for vulkan

renchenglei opened the pull request:
As vulkan library name has been changed from broxton to celadon,
we need also make related change for sepolicy.

Tracked-On: OAM-90956
Signed-off-by: Ren, Chenglei <chenglei.ren@intel.com>

sysopenci reviewed (commented):
Autobuild started from pull-request-changes on this PR.

FAILURE: CheckBug Bad comments/Bugs

For more information, see: /absp/builders/celadon-autobuild/builds/2011

renchenglei committed:
Adjust sepolicy change for vulkan

As vulkan library name has been changed from broxton to celadon,
we need also make related change for sepolicy.

Tracked-On: OAM-90956
Signed-off-by: Ren, Chenglei <chenglei.ren@intel.com>

renchenglei commented:
@feijiang1 & @YuanjunHuang, could you help take a review of this changes? This is to help fix all vulkan CTS broken issue since weekly 18 build.

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckApprovals Some checks are failed. See the step results.html

More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/1876

renchenglei commented:
Also need domain owner's approval. @ceiba1985 & @kwang13, could you also help take a review of this changes? Thanks a lot! :)

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckApprovals Some checks are failed. See the step results.html

More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/1877

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckBug Bad comments/Bugs

More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/1878

sysopenci reviewed (commented):
PR integrated into 'preintegration' builder



More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/1879

Done by "Ren, Chenglei" <chenglei.ren@intel.com>

sysopenci reviewed (commented):
PR integrated into 'premerge' builder



More details, logs, and binaries: /absp/builders/celadon-premerge/builds/947

Done by "Ren, Chenglei" <chenglei.ren@intel.com>

sysopenci reviewed (commented):
cactus finished building and testing your changes 

It was requested for merge by "Ren, Chenglei" <chenglei.ren@intel.com> 

so we merge it. Thanks for your contribution.

more details, logs, and binaries: /absp/builders/celadon-mergerequest/builds/1644
